task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -1,13 +1,13 @@
----
```

Comment:
I should upload my wonderful tkn fmt tool, I was making some refinements and I think I broke something. With it this would be `diff -u <(tkn fmt <(curl -sL $MAIN)) <(tkn fmt <(curl -sL $PR))` or something along those lines...

Reply:
Dropping a link to https://github.com/zregvart/tkn-fmt